src/components/Table/Table.story.jsx

```diff
@@ -3812,4 +3812,27 @@ storiesOf('Watson IoT/Table', module)
         propTablesExclude: [StatefulTable],
       },
     }
-  );
+  )
```

Comment:
Since we have so many Table stories let's see if we can add this feature to an existing story

Reply:
> Since we have so many Table stories let's see if we can add this feature to an existing story

@davidicus Merged into the stateful table example with overflow menu, etc.